src/components/home/EventListElementComponent.tsx

```diff
@@ -58,18 +57,25 @@ const EventListElementComponent = (props: Props) => {
     }
   }
 
+
   return (
     <View style={{ marginVertical: 5, width: '90%' }}>
-      <Shadow startColor={Colors.dangerShadow} endColor={Colors.dangerShadow} distance={2}
-        offset={[0, 1]} paintInside={true} style={{ alignSelf: 'stretch', borderRadius: 5 }}>
-        <TouchableOpacity onPress={props.onPress} activeOpacity={0.8}>
-          <View style={Cs.unsentEventsContainer} >
-            <Text style={[Ts.eventListElementTitle, { color: Colors.dangerButton1 }]}>{title}</Text>
-            <Text style={Ts.eventListElementTextClear}>{displayDateTime()}</Text>
-            <Text style={Ts.eventListElementTextFaded}>{t('observations in list') + ': ' + observationCount() + ' ' + (observationCount() === 1 ? t('piece') : t('pieces'))}</Text>
-          </View>
-        </TouchableOpacity>
-      </Shadow>
+      <TouchableOpacity onPress={props.onPress} activeOpacity={0.8} style={{
```

Comment:
Container styles should be imported from /styles/ContainerStyles.tsx. Create a new style into /styles/ContainerStyles.tsx, for example "buttonShadowContainer". Import it like this: "import Cs from '../styles/ContainerStyles.tsx'". Repeat for different shadow styles, reuse if possible. If shadow styles differ by only one attribute, you can use 'style={[Cs.buttonShadowContainer, { shadowColor: Colors.dangerShadow }]}'.